### PR TITLE
feat(toolbar): added sticky

### LINKED ISSUE #4113: Toolbar: provide a way to make it sticky
In cockpit we use toolbars to filter really long lists, of hundreds of items. When the user scrolls down the list, they might still want to edit the filter options.

Could we consider making the toolbars optionally sticky? 

https://www.patternfly.org/v4/components/toolbar/

### PR DESCRIPTION
**Pull request by KKoukiou**
Fixes #4113

### CODE REVIEW

File: src/patternfly/components/Toolbar/examples/Toolbar.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Toolbar relies on groups (`.pf-c-toolbar__group`) and items (`.pf-c-toolbar__ite
 | `.pf-m-visible{-on-[breakpoint]}` | `.pf-c-toolbar > *` | Modifies toolbar element to be shown, at optional breakpoint. |
 | `.pf-m-align-right{-on-[breakpoint]}` | `.pf-c-toolbar > *` | Modifies toolbar element to align right, at optional breakpoint. |
 | `.pf-m-align-left{-on-[breakpoint]}` | `.pf-c-toolbar > *` | Modifies toolbar element to align left, at optional breakpoint. |
+| `.pf-m-sticky-top` | `.pf-c-toolbar-*` | Modifies toolbar element to be sticky to the top of its container. |
```
Comment (mcoker) [inline src/patternfly/components/Toolbar/examples/Toolbar.md:37]: Updated the class it applies to, though this table is for modifiers that apply to the toolbar items/children. Can you move this line down to the table under the "Overview" section? You can add this as the first modifier in that list (above the entry `.pf-m-expanded | .pf-c-toolbar__expandable-content |  Modifies expandable content section for the expanded state.`



```suggestion
| `.pf-m-sticky-top` | `.pf-c-toolbar` | Modifies toolbar element to be sticky to the top of its container. |
```

**Review by mcoker — CHANGES_REQUESTED** [review]
Looks great!! Thanks!! Just one small nit about the documentation.

File: src/patternfly/components/Toolbar/examples/Toolbar.md (hunk 1 above)
Comment (KKoukiou) [reply src/patternfly/components/Toolbar/examples/Toolbar.md]: Sure! Thanks for catching this.

File: src/patternfly/components/Toolbar/examples/Toolbar.md [hunk 2]
```diff
@@ -647,6 +647,7 @@ As the toolbar component is a hybrid layout and component, some of its elements
 | `.pf-c-toolbar__content` | `<div>` | Initiates a toolbar content container. **Required** |
 | `.pf-c-toolbar__content-section` | `<div>` | Initiates a toolbar content section. This is used to separate static elements from dynamic elements within a content container. There should be no more than one `.pf-c-toolbar__content-section` per `.pf-c-toolbar__content` **Required** |
 | `.pf-c-toolbar__expandable-content` | `<div>` | Initiates a toolbar expandable content section. |
+| `.pf-m-sticky-top` | `.pf-c-toolbar` | Modifies toolbar element to be sticky to the top of its container. |
```
Comment (mcoker) [inline src/patternfly/components/Toolbar/examples/Toolbar.md:650]: Missed this originally

```suggestion
| `.pf-m-sticky-top` | `.pf-c-toolbar` | Modifies toolbar component to be sticky to the top of its container. |
```

**Review by mcoker — CHANGES_REQUESTED** [review]
Sorry, I missed a small docs nit.

**Review by mcoker — APPROVED** [review]
LGTM! Thanks @KKoukiou!! 🥳

**Pull request merged**